charts/tfy-prometheus-config/values.yaml

```diff
@@ -262,6 +262,42 @@ serviceMonitors:
     namespaceSelector:
       matchNames:
         - elasti
+  # serviceMonitors.gpu Service monitor for gpu
+  gpu:
```

Comment:
Why can't this be part of tfy-gpu-operator chart itself?

https://github.com/truefoundry/infra-charts/blob/1896c76d55f8e7d92c0523922fc2857656715348/charts/tfy-gpu-operator/values.yaml#L293

Reply:
Before that,  is this servicemonitor for operator controller or for dcgm daemonsets (which actually power all the gpu metrics) ?

Reply:
this is for dcgm